automated install/basic-install.sh

```diff
@@ -2323,34 +2323,6 @@ main() {
     # the service before enabling causes installer to exit
     enable_service pihole-FTL
 
-    # If this is an update from a previous Pi-hole installation
-    # we need to move any existing `pihole*` logs from `/var/log` to `/var/log/pihole`
-    # if /var/log/pihole.log is not a symlink (set during FTL startup) move the files
-    # can be removed with Pi-hole v6.0
-    # To be sure FTL is not running when we move the files we explicitly stop it here
-
-    stop_service pihole-FTL &>/dev/null
-
-    if [ ! -d /var/log/pihole/ ]; then
-        mkdir -m 0755 /var/log/pihole/
-    fi
```

Comment:
Do we need to keep this? I have not found any other location where we are creating the directory (e.g. for fresh installs).

Reply:
Look here: https://github.com/pi-hole/pi-hole/blob/67de7be802d942947dc1d401de7527e3f8605923/advanced/Templates/pihole-FTL-prestart.sh#L14

Reply:
Ah, yeah, my mistake. I `grep`ped for `/var/log/pihole/` including a trailing slash which didn't reveal this one.